kaltura-common/src/rxjs/operators/cancel-on-destroy.ts

```diff
@@ -0,0 +1,50 @@
+import { Observable } from 'rxjs/Observable';
```

Comment:
creation method should be imported like so:
``` import { Observable, Subject, asapScheduler, pipe, of, from, interval, merge, fromEvent } from 'rxjs'; ```
see:
https://github.com/ReactiveX/rxjs/blob/master/docs_app/content/guide/v6/migration.md#import-paths

Reply:
correcting my previous comment.

Reply:
`merge` is not available for import from 'rxjs' in v5.5.6 yet

Reply:
I see, I thought this work was done as part of v6 migration. my bad.